### ISSUE TITLE
Save relative URLs of images in database

### ISSUE THREAD

**Issue by micka-fdz** [PrestaSafe/prettyblocks#52]
We need to save relative URLs instead of absolute URLs.

**Comment by bc-devle** [comment]
Agreed, with full URLs we have to replace domain / subdomain text in database when we change (I did it earlier).

in the meantime here are some requests to do it :
update xt_configuration set value = replace(value, 'old_domain', 'new_domain');
update xt_prettyblocks_lang set state = replace(state, 'old_domain', 'new_domain');

**Comment by micka-fdz** [comment]
After a discussion with @jf-viguier, we think it would be necessary to store only files path in DB.
URLs can be easily calculated using PrestaShop methods.

**Comment by PrestaSafe** [comment]
Hi everyone, yes we will change that, but we need to think of all the problems it could generate (prestashop in sub folder, multishop multilang etc...) We will probably use the HelperBuilder with the : `$/path/to/img` for generating the url. 
A lot of tests should be made
